cv/qc-cv.obo

```diff
@@ -539,14 +564,1316 @@ is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+synonym: "IS-3C"  RELATED []
+
+[Term]
+id: QC:4000182
+name: Mean precursor charge in all MS2
+def: "Mean precursor charge in all MS2" [PSI:QC]
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+
+[Term]
+id: QC:4000183
+name: Median precursor charge in all MS2
+def: "Median precursor charge in all MS2" [PSI:QC]
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+
+[Term]
+id: QC:4000184
+name: Number of different proteins from all PSM
+def: "Number of different protein from all PSM" [PSI:QC]
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+property_value: has_units UO:0000189
+
+[Term]
+id: QC:4000185
+name: Number of identified proteins
+def: "Number of identified proteins at given FDR threshold, first number is the number of proteins (considering sequence only), second number is the FDR threshold applied (negative if no threshold applied)" [PSI:QC]
+comment: The number of identifications are likely to only make local sense within a confined set of identification parameters (identification algorithm, searchspace, FDR method).
+is_a: QC:4000004 ! n-tuple
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+property_value: has_units UO:0000189
+
+[Term]
+id: QC:4000186
+name: Total number of PSM
+def: "Total number of PSM" [PSI:QC]
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+property_value: has_units UO:0000189
+
+[Term]
+id: QC:4000187
+name: Number of identified peptides
+def: "Number of identified peptides at given FDR threshold, first number is the number of peptides (considering sequence only), second number is the FDR threshold applied (negative if no threshold applied)" [PSI:QC]
+comment: The number of identifications are likely to only make local sense within a confined set of identification parameters (identification algorithm, searchspace, FDR method).
+is_a: QC:4000004 ! n-tuple
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+property_value: has_units UO:0000189
+
+[Term]
+id: QC:4000188
+name: Number of identified spectra
+def: "Number of identified spectra at given FDR threshold, first number is the number of spectra, second number is the FDR threshold applied (negative if no threshold applied)" [PSI:QC]
+comment: The number of identifications are likely to only make local sense within a confined set of identification parameters (identification algorithm, searchspace, FDR method).
+is_a: QC:4000004 ! n-tuple
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+property_value: has_units UO:0000189
+
+[Term]
+id: QC:4000189
+name: ID ratio
+def: "The ratio of identified and recorded spectra." [PSI:QC]
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+property_value: has_units STATO:0000300
+
+[Term]
+id: QC:4000190
+name: Precursor errors (Da) Q1, Q2, Q3
```

Comment:
there are at least two ways how this could be interpreted:
1) mass deviation of PC to identified peptide
2) mass deviation of PC when just correcting the isotope peak its pointing to (without doing any ID).

there is probably more ways...

Reply:
so there is a set of terms for ppm mass difference (identified and precursor), so I think this should be mass difference in Da just to balance.

Reply:
I made changes there